src/game/level.rs

```diff
@@ -333,12 +373,33 @@ impl<'a, 'b: 'a> LevelBuilder<'a, 'b> {
         };
         let half_width = bounds.size[0] / 100.0 * 0.5;
 
-        self.scene.decors_buffer()
-            .push(&low, -half_width, 0.0, bounds.size[1], bounds, sector.light_info)
-            .push(&low, half_width, bounds.size[0], bounds.size[1], bounds, sector.light_info)
+        self.scene
+            .decors_buffer()
+            .push(&low,
+                  -half_width,
+                  0.0,
+                  bounds.size[1],
+                  bounds,
+                  sector.light_info)
+            .push(&low,
+                  half_width,
+                  bounds.size[0],
+                  bounds.size[1],
+                  bounds,
+                  sector.light_info)
             .push(&high, -half_width, 0.0, 0.0, bounds, sector.light_info)
-            .push(&low, half_width, bounds.size[0], bounds.size[1], bounds, sector.light_info)
-            .push(&high, half_width, bounds.size[0], 0.0, bounds, sector.light_info)
+            .push(&low,
+                  half_width,
+                  bounds.size[0],
+                  bounds.size[1],
+                  bounds,
+                  sector.light_info)
+            .push(&high,
+                  half_width,
+                  bounds.size[0],
+                  0.0,
+                  bounds,
+                  sector.light_info)
```

Comment:
I find the newlines above make the code much harder to read. I know that rustfmt breaks function calls even when they fit in the width, but I think this should be balanced by the number of args: i.e. if it would cost 8 newlines to break a line which fits in 100 chars, it should prefer to keep it.